web/client/components/background/PreviewIcon.jsx

```diff
@@ -46,7 +46,7 @@ class PreviewIcon extends React.Component {
         const compatibleWmts = this.props.layer.type === "wmts" && has(this.props.layer.allowedSRS, this.props.projection);
         const containerClass = this.props.vertical ? 'background-preview-icon-container-vertical' : 'background-preview-icon-container-horizontal';
         const type = this.props.layer.visibility ? ' bg-primary' : ' bg-body';
-        const valid = ((validCrs || compatibleWmts || this.props.layer.type === "wms" || this.props.layer.type === "empty") && !this.props.layer.invalid );
+        const valid = ((validCrs || compatibleWmts || ["wms", "empty", "osm"].includes(this.props.layer.type)) && !this.props.layer.invalid );
```

Comment:
includes is not supported by every browser, can you use a lodash function?

Reply:
osm is valid in every CRS ?

Reply:
Since we would have already specified the projection defs for the custom CRS, openlayer should be able to reproject, if I'm being right. Also this would help us select and deselect the OSM background, as of now we can enable but not re-select from other background.